go/mysql/mysqldbconn/conn.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2012, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mysqldbconn
+
+import (
+	"github.com/youtube/vitess/go/mysql"
+	"github.com/youtube/vitess/go/sqldbconn"
+)
+
+func init() {
+	sqldbconn.NewSqlDBConn = mysql.Connect
```

Comment:
This could be the default for all programs, and just the unit tests override it when they need to?